test/functional/multiwallet.py

```diff
@@ -18,6 +20,21 @@ def __init__(self):
         self.extra_args = [['-wallet=w1', '-wallet=w2', '-wallet=w3']]
 
     def run_test(self):
+        self.stop_node(0)
```

Comment:
Note for other reviewers: It seemed slightly weird to start-stop the node at the beginning of this test. The reason we need to do that is that the `w1` file doesn't exist until the node has been started, and we can't do the symlink test below without an existing wallet file.

micronit: I think it feels more natural to do these invalid tests at the end of this test script, and removes the confusing start-stop at the beginning of the test. That's just personal taste though.